src/java/main/org/apache/zookeeper/server/ZooKeeperServer.java

```diff
@@ -542,7 +546,8 @@ public boolean isRunning() {
     }
 
     public void shutdown() {
-        shutdown(false);
+        // switch to false when ZOOKEEPER-2845 is fixed.
```

Comment:
make more personalized comment so that we can find that it's dropbox-specific patch?